std_msgs/Header header                  # copied from the PC2
uint32 n_obstacles                      # num of detected obstacles in the PC2
emulated_srs/Obstacle[] data
